doc/modules/cassandra/pages/new/virtualtables.adoc

```diff
@@ -170,6 +173,22 @@ counters |       26214400 |           0 |         0 |       NaN |
 (4 rows)
 ....
 
+=== CQL metrics Virtual Table
+
+Lists CQL prepared statements metrics. A query on CQL metrics virtual table lists below metrics.
```

Comment:
Same comment as before. If we opt out from duplication of certain info about the metrics (that we have already on the metrics page), I would say at least I prefer to say `Metrics specific to CQL prepared statement caching.`

Reply:
Done